pkg/cmd/root.go

```diff
@@ -135,7 +135,7 @@ func NewCmd(rootCmd *cobra.Command, exeName, desc string, initConfigHandler Init
 
 // Add the command line properties for the logger and path config
 func (c *agentRootCommand) addBaseProps() {
-	c.props.AddStringPersistentFlag(PathConfigFlag, ".", "Configuration file path for the agent")
+	c.props.AddStringPersistentFlag(PathConfigFlag, ".", "YAML configuration file path for the agent")
```

Comment:
Is it confusing to say "YAML configuration.." When this is just the path to where the YAML file is, not the path including the YAML filename?

Reply:
ah. Maybe "Path to the YAML configuration file for the agent"?

Reply:
Path to directory containing YAML...?

Reply:
ok